### ISSUE TITLE
Unable to run Cloud Run locally due to invalid gcloud command

### ISSUE THREAD

**Issue by TheBestMoshe** [GoogleCloudPlatform/cloud-code-vscode#301]
**Environment**:
Extension version: 1.5.2
VSCode version: 1.48.0
OS: darwin

**Description**:
`gcloud` command does not match the latest version of `gcloud`

**Repro step**:
When I run `Cloud Code: Run on Cloud Run Emulator` from the VSCode command pallet, it throws an error (`main_api` is the image name):
```
Running "Cloud Run: Run/Debug Locally" configuration (.vscode/launch.json) in Run mode

To view more detailed logs, go to Output channel : "Cloud Run: Run/Debug Locally - Detailed"

Dependency check started
Dependency check succeeded
ERROR: (gcloud.alpha.code.export) unrecognized arguments:
  --image-name (did you mean '--image'?)
  main_api
  --memory-limit (did you mean '--memory'?)
  256Mi
  To search the help text of gcloud commands, run:
  gcloud help -- SEARCH_TERMS
```

When I run `gcloud alpha code export --help` I get this results:
```
SYNOPSIS
    gcloud alpha code export [--cloudsql-instances=[CLOUDSQL_INSTANCE,...]]
        [--cpu=CPU] [--image=IMAGE]
        [--kubernetes-file=KUBERNETES_FILE; default="pods_and_services.yaml"]
        [--local-port=LOCAL_PORT] [--memory=MEMORY]
        [--service-name=SERVICE_NAME] [--source=SOURCE]
        [--appengine | --builder=BUILDER
          | --dockerfile=DOCKERFILE; default="Dockerfile"]
        [--application-default-credential | --service-account=SERVICE_ACCOUNT]
        [--env-vars=[KEY=VALUE,...] | --env-vars-file=FILE_PATH]
        [--skaffold-file=SKAFFOLD_FILE; default="skaffold.yaml"
          | --no-skaffold-file] [GCLOUD_WIDE_FLAG ...]
```

`gcloud version`:
```
Google Cloud SDK 312.0.0
alpha 2020.09.25
bq 2.0.61
core 2020.09.25
gsutil 4.53
```

It seems like there is a mismatch between the extension and the `gcloud` version installed on my system.

**Comment by sujit-kamireddy** [comment]
Hi, we released cloud code v1.6 yesterday which should fix this issue. 
Can you try your scenario with the latest version of cloud code?

**Comment by TheBestMoshe** [comment]
Thanks, it fixed the problem.
